app/models/container.rb

```diff
@@ -9,6 +9,7 @@ class Container < ActiveRecord::Base
   belongs_to :container_definition
   belongs_to :container_image
   has_one    :container_image_registry, :through => :container_image
+  has_one    :container_security_context, :through => :container_definition
```

Comment:
@zakiva don't treat this as a relationship. Add the columns to the existing `container_definition` table.